Dangerfile

```diff
@@ -1,79 +1,82 @@
-message "@#{github.pr_author} Thank you for your pull request! I'm here to help with some tips and recommendations. Please take a look at the list provided and help us review and accept your contribution! And **don't be discouraged if you see errors** -- we're here to help."
+# run only on system tests? This will break interactions with junit test reports but fix automated screenshot posting 
+if ENV['TEST_TYPE'] == "system"
```

Comment:
I'm guessing this is not working.

Reply:
Ah ok actually the Ruby code is working; the insertion of the ENV variable is not. Maybe we have to turn that on in Travis...